resources/lang/en.js

```diff
@@ -7,42 +7,42 @@ class CactbotLanguageEn extends CactbotLanguage {
 
   InitStrings(playerName) {
     this.kAbility = Object.freeze({
-      DragonKick: 'Dragon Kick',
-      TwinSnakes: 'Twin Snakes',
-      Demolish: 'Demolish',
-      Verstone: 'Verstone',
-      Verfire: 'Verfire',
-      Veraero: 'Veraero',
-      Verthunder: 'Verthunder',
-      Verholy: 'Verholy',
-      Verflare: 'Verflare',
-      Jolt2: 'Jolt II',
-      Jolt: 'Jolt',
-      Impact: 'Impact',
-      Scatter: 'Scatter',
-      Vercure: 'Vercure',
-      Verraise: 'Verraise',
-      Riposte: 'Riposte',
-      Zwerchhau: 'Zwerchhau',
-      Redoublement: 'Redoublement',
-      Moulinet: 'Moulinet',
-      EnchantedRiposte: 'Enchanted Riposte',
-      EnchantedZwerchhau: 'Enchanted Zwerchhau',
-      EnchantedRedoublement: 'Enchanted Redoublement',
-      EnchantedMoulinet: 'Enchanted Moulinet',
-      Tomahawk: 'Tomahawk',
-      Overpower: 'Overpower',
-      HeavySwing: 'Heavy Swing',
-      SkullSunder: 'Skull Sunder',
-      ButchersBlock: "Butcher's Block",
-      Maim: 'Maim',
-      StormsEye: "Storm's Eye",
-      StormsPath: "Storm's Path",
-      TrickAttack: 'Trick Attack',
-      Embolden: 'Embolden',
-      Aetherflow: 'Aetherflow',
-      ChainStrategem: 'Chain Strategem',
-      Hypercharge: 'Hypercharge',
+      DragonKick: 'Dragon Kick', // 0x4a
```

Comment:
Were these id comments useful for you when translating? I feel like these ids are all already inside of lang.js and so duplicating them here is a bit noisy.  I think I'd prefer to not have them if it was just me, but if they were useful in translating, then I'm happy to leave them.

Reply:
Translators can reference this ID to query XIVDB (`0x8c` -> `/status/140`), or extracted FFXIV data itself.

Reply:
Oh, it was duplicate, I didn't saw them. I'll remove these comments.